### DISCUSSION TITLE
Keep camera on after scanning QR code?

### DISCUSSION THREAD

**Discussion by Legrosbob** [gopro/labs#1188] [Q&A] [answered]
Hi,

Might be a basic question, but I can't find any info about my issue.

Is there a way to keep my GoPro on for some time after scanning a Labs QR code?

I'd like to scan the code, then be able to see the screen while I position the camera, either directly on the LCD or through Quik. I could then set a command to turn off the camera after say, 5 minutes.

The camera turns off as soon as I scan the code and disconnects from Quik.

Thanks!

**Comment by dnewman-gpsw** [comment]
All depends on what you are scanning, and it will only turn off if that is what you told it to do. Settings will not turn off the camera. If you want it to turn off after five minutes, you can script it do to that, although not recommended if your script is a battery saving time wake event. Or point the camera first, then scan your code.

**Comment by Legrosbob** [comment]
To be more specific, I'm trying to film road intersections for traffic counting. The gopro will be at the end of an extension pole attached to a light post, and I need to adjust the direction that the gopro is pointing once it is mounted on the post. I need to scan the code, mount the camera, then connect to Quik to see what the camera is seeing. Worst case scenario I could use my phone camera to zoom in on the front LCD and adjust the direction this way.

What I'm trying to script is pretty simple. I want to do multiple scheduled captures throughout the day. For example this:

![Screenshot_20241128_220141_Gmail](https://github.com/user-attachments/assets/ac25c149-6641-4cea-a44d-1bdb9b2e5ba7)
!07:00N!S!7200E!16:00N!S!7200E

However, as soon as I scan the code, the camera turns off and disconnects from Quik. How can I script so that the camera stays on for a few minutes after I scan the code? Battery life is not an issue since I'll be on external battery.

Thanks!

**Reply by dnewman-gpsw** [reply]
Why make it sleep until 7am as its first action?  If you make it the last action, it will record first and stay on.

**Reply by dnewman-gpsw** [reply]
There are so many ways to address this.

**Reply by dnewman-gpsw** [reply, marked as answer]
scan *QRDR=1 once. This is needed to enable scanning QR Codes while using Quik Preview, most Labs projects don't use Quik.

After scan the above. Enable Quik Preview. Then finally scan this:

```>r:A"App Live"+!R<07:00!07:00N>18:00!07:00N<09:00!S+!09:00E+!2N!16:00N!S!18:00E!2O```

How it works:

```
>r:A - Is the App connected and Previewing
"App Live"+!R - Print "App Live" and repeat
<07:00!07:00N - if less than 7am sleep until 7am
>18:00!07:00N - if greater than 6pm sleep until 7am
<09:00!S+!09:00E+!2N - if less than 9am, record until 9am, pause 2s (needed on some models)
!16:00N!S!18:00E!2O - sleep until 4pm, record until 6pm, Off in two seconds.
```

The script will start when you quit the Quik app (I have confirmed on Android only.)
